fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -183,6 +183,9 @@
     @Nonnull
     protected final RecordMetaDataProvider metaDataProvider;
 
+    @Nullable
+    protected final Descriptors.FileDescriptor localMetaDataFileDescriptor;
```

Comment:
Why does the record store need this? Isn't this included in the `RecordType` information in the `RecordMetaData`?

Reply:
I proposed adding it to the record store builder, but I don't think it should be in the record store itself. I think on the builders, it should be explicitly set (as it was before), though I can kind of get behind using the record meta-data if possible.

Reply:
dropped it.